lib/src/utils/constants.dart

```diff
@@ -0,0 +1,8 @@
+import 'package:flutter/material.dart';
+
+class Constants{
+  static final widthHeightOfDice = 100.0;
+  static final diceAmount = 6;
+  static final textHeight1_4 = 1.4;
+  static final fontWeight400 = FontWeight.w400;
+}
```

Comment:
change static final for static const and change to abstract class

Reply:
Done